airbyte-webapp/src/components/EntityTable/components/ChangesStatusIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classnames from "classnames";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Tooltip } from "components/ui/Tooltip";
+
+import { SchemaChange } from "core/request/AirbyteClient";
+
+import styles from "./ChangesStatusIcon.module.scss";
+
+interface ChangesStatusIconProps {
+  schemaChange?: SchemaChange;
+}
+
+export const ChangesStatusIcon: React.FC<ChangesStatusIconProps> = ({ schemaChange = "no_change" }) => {
+  if (schemaChange === "no_change") {
+    return null;
+  }
+  const iconStyle = classnames(styles.changesIcon, {
+    [styles.breaking]: schemaChange === "breaking",
+    [styles.nonBreaking]: schemaChange === "non_breaking",
+  });
+  let schemaChangeLocalizationKey;
+  switch (schemaChange) {
+    case "breaking":
+      schemaChangeLocalizationKey = "breaking";
+      break;
+    case "non_breaking":
+      schemaChangeLocalizationKey = "nonBreaking";
+      break;
+  }
+  const tooltipComponent = <FormattedMessage id={`connection.schemaChange.${schemaChangeLocalizationKey}`} />;
+  return (
+    <Tooltip
+      containerClassName={styles.tooltipContainer}
+      control={<FontAwesomeIcon className={iconStyle} icon={faExclamationCircle} size="2x" />}
+    >
+      {tooltipComponent}
```

Comment:
This tooltip bit can be cut way down.
```
<FormattedMessage id={`connection.schemaChange.${schemaChange === "non_breaking" ? "nonBreaking" : schemaChange}`} />
```
Or even just run the `schemaChange` through https://stackoverflow.com/a/61375162

Reply:
thanks for the utility function idea. added to preject